ui/components/graphql/subscriptions/ConfigurationSubscription.js

```diff
@@ -47,6 +47,7 @@ const configurationSubscription = graphql`
           id
           name
           filter_file
+          filter_resource
```

Comment:
`filter_resource` contains configYaml?

Reply:
Yes. filter_resource has an object and in that we have a key called `config` which contains the configYaml.